app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,35 @@
+class BookingsController < ApplicationController
+
+  def show
+    @booking = Booking.find(params[:id])
+  end
+
+  def new
+    @garden = Garden.find(params[:id])
+    @booking = Booking.new
+  end
+
+
+
+  def create
+    booking = Booking.new(booking_params)
+    garden = Garden.find(params[:id])
+    booking.garden = garden
+    booking.save
+
+    # if booking.save // else redirect_to 'bookings/new'_path(:booking)
+  end
+
+  private
+
+  def booking_params
+    params.require(:booking).permit(:start_date, :end_date)
+  end
+end
+
+    @review = Review.new(review_params)
```

Comment:
@georgkloeck - breaks the app - it's random text at the bottom of a file, not in a class.  Did you fire up the app and look in the browser?  If so, you would have found out it wouldn't render.

Reply:
lets just comment it out for now